test/test_runner.go

```diff
@@ -678,6 +683,25 @@ func (r *TestRunner) parseAndCheckImport(
 
 	env.CheckerConfig.ContractValueHandler = contractValueHandler
 
+	if location == BlockchainHelpersLocation {
```

Comment:
Instead of special casing it here, can we do it at the `(r *TestRunner) checkerImportHandler()` ?
e.g. Add one more case for the switch `case stdlib.BlockchainHelpersLocation:`

Reply:
Good idea :100: Fixed in https://github.com/onflow/cadence-tools/pull/181/commits/a109ffffae72e4a49b3b5ab1e8528408eb2aeae2. I also had to update the `interpreterImportHandler` switch.